local_server.sh

```diff
@@ -25,6 +25,9 @@ redis-server config/redis_local.conf &
 echo "Starting compass watch"
 compass watch server &
 
+echo "Starting jsx compiler"
+jsx --watch -x jsx server/static/jsx/ server/static/js/ &
```

Comment:
Good point on the global vs local installation, I'll make sure to change that. 

In terms of prod (minified) vs dev version of React, do you know of a clean way to serve the correct file based on the environment? Or should I just use an environment variable in the template?